src/azure-cli/azure/cli/command_modules/acr/check_health.py

```diff
@@ -26,7 +26,8 @@
 RECOMMENDED_NOTARY_VERSION = "0.6.0"
 NOTARY_VERSION_REGEX = re.compile(r'Version:\s+([.\d]+)')
 DOCKER_PULL_WRONG_PLATFORM = 'cannot be used on this platform'
-
+IMAGE_INPUT_ERROR_MESSAGE = "The specified image tag was not found. Please check image name for --image."
```

Comment:
Not a huge fan of "Please check image name for --image." Can we word this better? Same with repository below.

Reply:
After consulting with Jason and Roy, I will be implementing this differently to say that image is not found, and then suggest an image they might have intended to enter. This gives helpful specific information in case user entered image name incorrectly with typo or accidentally pasted the wrong image name. If incorrect repo is entered, a repo not found error will come from another method and error message will come from the backend, I realized I don't need to check for it again.